core/store/src/lib.rs

```diff
@@ -73,14 +101,58 @@ impl Store {
         (dir, opener)
     }
 
-    pub(crate) fn new(storage: Arc<dyn Database>) -> Store {
-        Store { storage }
+    /// Constructs new object backed by given database.
+    ///
+    /// Note that you most likely don’t want to use this method.  If you’re
+    /// opening an on-disk storage, you want to use [`Self::opener`] instead
+    /// which takes scare of opening the on-disk database and applying all the
+    /// necessary configuration.  If you need an in-memory database for testing,
+    /// you want either [`crate::test_utils::create_test_node_storage`] or
+    /// possibly [`crate::test_utils::create_test_store`] (depending whether you
+    /// need [`NodeStorage`] or [`Store`] object.
+    pub fn new(storage: Arc<dyn Database>) -> Self {
+        Self { storage }
+    }
+
+    pub fn get_store(&self, temp: Temperature) -> Store {
```

Comment:
This is the main function that people should call? (if yes -add a small comment)